docker/build.sh

```diff
@@ -15,8 +15,8 @@ HELP_TEXT="$(cat <<EOF
    jar : build Leonardo jar
    -d | --docker : (default: no action) provide either "build" or "push" to
            build or push a docker image.  "push" will also perform build.
-   -r | --registry: (default: dockerhub) can be either "dockerhub" or "gcr".
-           Users of gcr should have the gcloud tool installed and configured.
+   -dr | --dockerhub-registry: The dockerhub registry to push to
+   -gr | --gcr-registry: The GCR registry to push to
    -p | --project: set the project used at either dockerhub or with gcr
```

Comment:
I don't think `-p` and `$DOCKER_PROJECT` are used anymore now that the docker/gcr repos are specified explicitly. Can we remove it?

Reply:
yep